interface/themes/style_light.css

```diff
@@ -1161,3 +1161,7 @@ div.notab-right {
 .report_search_bar {
   background-color: #EADDB4;
 }
+
+.iziModal.isFullscreen .iziModal-iframe {
+  height: 100% !important;
```

Comment:
W00t this looks more better and cleaner. Thanks for the changes @d3n23z . Keep them coming